### PR TITLE
add bar model

### PR DESCRIPTION
**Pull request by jjkazm**
add bar model; final styling will be added later

### CODE REVIEW

File: app/models/bar.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Bar < ApplicationRecord
+end
```
Comment (wbucko) [inline app/models/bar.rb:2]: Consider adding validations to Bar. If you do so, please add specs as well.

**Review by wbucko — COMMENTED** [review]
I'm not big fan of a generators. Next time consider writing thins on your own :)

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  padding: 0 5px 7px;
+}
+
+div {
+  &.field, &.actions {
+    margin-bottom: 10px;
+  }
+}
+
+#notice {
```
Comment (brac193) [inline app/assets/stylesheets/scaffolds.scss:49]: general remark - don't use id for styling https://dev.to/claireparker/reasons-not-to-use-ids-in-css-4ni4

File: app/controllers/bars_controller.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  # POST /bars
+  # POST /bars.json
+  def create
+    @bar = Bar.new(bar_params)
+
+    respond_to do |format|
+      if @bar.save
+        format.html { redirect_to @bar, notice: 'Bar was successfully created.' }
+        format.json { render :show, status: :created, location: @bar }
```
Comment (brac193) [inline app/controllers/bars_controller.rb:32]: do we need json?

**Review by wbucko — APPROVED** [review]

**Review by brac193 — APPROVED** [review]

**Pull request merged**